go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/openshift/cluster-api-provider-packet
+
+go 1.13
+
+require (
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/go-logr/logr v0.2.1
+	github.com/openshift/api v0.0.0-20200901182017-7ac89ba6b971
+	github.com/openshift/machine-api-operator v0.2.1-0.20201002104344-6abfb5440597
+
+	// kube 1.18
+	k8s.io/api v0.19.0
+	k8s.io/apimachinery v0.19.0
+	k8s.io/client-go v0.19.0
+	k8s.io/klog/v2 v2.3.0
+	sigs.k8s.io/controller-runtime v0.6.2
+	sigs.k8s.io/controller-tools v0.3.0
+	sigs.k8s.io/yaml v1.2.0
+)
+
+replace (
+	sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20200929152424-eab2e087f366
```

Comment:
I'm curious, are these modules actually needed in this provider?  Many of the vendored files are related to these clouds.

Reply:
Following up on this, it looks like they are pulled in from the machine-api-operator dependency and causes failures if not specified.